modules/search/class.jetpack-search.php

```diff
@@ -862,11 +896,17 @@ public function add_post_type_aggregation_to_es_query_builder( array $aggregatio
 	 * @param Jetpack_WPES_Query_Builder $builder The builder instance that is creating the ES query
 	 */
 	public function add_date_histogram_aggregation_to_es_query_builder( array $aggregation, $label, Jetpack_WPES_Query_Builder $builder ) {
+		$args = array(
+			'interval' => $aggregation['interval'],
+			'field'    => ( ! empty( $aggregation['field'] ) && 'post_date_gmt' == $aggregation['field'] ) ? 'date_gmt' : 'date',
+		);
+
+		if ( isset( $aggregation['min_doc_count'] ) ) {
+			$args['min_doc_count'] = intval( $aggregation['min_doc_count'] );
```

Comment:
I didn't know this existed! Let's make 1 be the default.

Reply:
Done